tests/integration_tests/security/test_jail.py

```diff
@@ -514,42 +509,6 @@ def test_args_resource_limits(test_microvm_with_api):
     check_limits(pid, NOFILE, FSIZE)
 
 
-def test_negative_file_size_limit(uvm_plain):
```

Comment:
There couple of issues with this test. First, there is no way to test `negative` case now. If VM uses 128MB of memory,
than limit needs to be 128MB. And when we create a snapshot it will create 128MB mem file. So there is no way for us to test this with FC. Second, what is the reason for this test overall? If FC somehow is compromized and asked to 
create a bunch of files, what prevents it to create a lot of small files and never hit the file size limit?

Reply:
Oh ok. I thought the rlimit was referring to the total amount of file sizes, not the size of each file individually.


> If FC somehow is compromized and asked to create a bunch of files, what prevents it to create a lot of small files and never hit the file size limit?

There is also a limit in the maximum amount of open file descriptors, so that covers that.

So, I think it's worth to keep the test and set the limit to something smaller than then memory of the guest. This should fail.

Reply:
Ok, redone the commit. Now test just tries to start VM to verify the limit.